sky/shell/platform/ios/semantics_listener_impl.h

```diff
@@ -0,0 +1,101 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef SKY_SHELL_PLATFORM_IOS_SEMANTICS_LISTENER_IMPL_H_
+#define SKY_SHELL_PLATFORM_IOS_SEMANTICS_LISTENER_IMPL_H_
+
+#include <map>
```

Comment:
<shrug> ... we have a few of them in our ios/mac codebase, and Chinmay saw me using one without commenting on it, but I can verify with him when he gets back.

Out of curiosity, what would the alternative here be to using stdlib?